submissions/bitwuzla.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "Bitwuzla",
+  "contributors": ["Aina Niemetz", "Mathias Preiner"],
+  "contacts": ["Mathias Preiner <preiner@cs.stanford.edu>"],
+
+  "archive": {
+    "url": "https://zenodo.org/records/11288326/files/bitwuzla-submission-smtcomp-2024.zip?download=1",
+    "h": {"sha256": "2396c5dbb5a377e9b399a688d711918004cfea83e1f7a6f4e7bf8eb9b369854a"}
+  },
+
+  "command": ["bin/bitwuzla"],
+
+  "website": "https://bitwuzla.github.io",
+  "system_description": "https://bitwuzla.github.io/data/smtcomp2024/paper.pdf",
+  "solver_type": "Standalone",
+  "seed": "42",
+  "participations": [
+    {
+      "tracks": ["SingleQuery"],
+      "logics": "ˆ(QF_)?(A)?(UF)?(BV|FP|FPLRA)+$",
```

Comment:
@mpreiner The regular expressions do not match any of the logics because of the weird unicode symbol `ˆ` instead of `^`. Can you fix it in the pull request?

I fixed that locally and tried running the current submission on our competition infrastructure with some trivial benchmarks. Everything else seems to be working so far!

Reply:
Ah yes, should not have copied this from the PDF... Should be fixed now.